src/actinia_core/resources/common/config.py

```diff
@@ -172,9 +172,12 @@ def __init__(self):
         # The directory to store temporary files
         self.TMP_WORKDIR = "%s/actinia/workspace/tmp" % home
         self.SECRET_KEY = "This is a very secret key that is used to sign tokens"
-        self.DOWNLOAD_CACHE = "/tmp/download_cache"  # The directory to cache downloaded data
-        self.DOWNLOAD_CACHE_QUOTA = 100  # The quota of the download cache in Gigibit
-        self.SAVE_INTERIM_RESULTS = False # If True the interim results (temporary mapset) are saved
+        # The directory to cache downloaded data
+        self.DOWNLOAD_CACHE = "/tmp/download_cache"
+        # The quota of the download cache in Gigibit
```

Comment:
Gigibit?

Reply:
I suppose it should indeed read Gig**a**bit (as being based on 1024 and not 1000), see:

https://github.com/mundialis/actinia_core/blob/95132551ed785ce498ebe90151363253c1fd9618/src/actinia_core/resources/download_cache_management.py#L144